js/src/personal/riyuan/components/about/About.tsx

```diff
@@ -1,28 +1,94 @@
+import aboutStyles from '@/personal/riyuan/components/about/About.module.css'
 
-export function About() {
+// Define an interface for the props
+interface InfoSectionProps {
+  title: string
+  description: string
+  listItems: string[]
+}
+// Function to render the info section
+function InfoSection({ title, description, listItems }: InfoSectionProps) {
   return (
-    <div id={'About'}>
-      <Title
-        className={styles.title}
-        ta="center"
-        mt={100}
-      >
-        {'About Page'}
-      </Title>
+    <div className={aboutStyles.info}>
+      <Title className={aboutStyles.title}>{title}</Title>
       <Text
         c="dimmed"
-        ta="center"
+        ta="left"
         size="lg"
-        maw={580}
         mx="auto"
         mt="xl"
       >
-        {
-          "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."
-        }
+        {description}
       </Text>
+      <List
+        mt={30}
+        spacing="sm"
+        size="sm"
+      >
+        {listItems.map((item, index) => (
+          <ListItem key={index}>{item}</ListItem>
+        ))}
+      </List>
+    </div>
+  )
+}
+
+export function About() {
+  const largeScreen = useMediaQuery('(min-width: 60em)')
+
+  const listItems = [
+    'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
+    'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
+    'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
+    'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
+  ]
+
+  const description =
+    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."
+
+  return (
+    <div
+      id={'About'}
```

Comment:
https://stackoverflow.com/a/73401865
Apparently, we can use IDs directly too like we do classNames from the CSS module.

Reply:
here, the classname is a global item under my website, and id is just ref link

Reply:
makes sense. In general, when we can we should try to avoid repeatedly typing the same strings like this "About" for the anchor and the div. One way to do this is to have a dummy CSS ID type and use `styles.About` or a constants file. 

Don't have to make any changes now though.